src/sass/vre.scss

```diff
@@ -6,3 +6,11 @@
 @import "modules/m-modal";
 @import "modules/m-omb-info";
 @import "modules/m-form-confirmation";
+
+.schemaform-first-field .schemaform-label, .schemaform-field-template > .usa-input-error {
+  margin-top: 0;
+}
+
+.schemaform-field-container {
```

Comment:
These changes are causing some spacing issues on other pages (like military history)

Reply:
Thanks, this has been removed, and the name field has been assigned the schemform-field-template class instead.